src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```diff
@@ -115,6 +115,19 @@ payum:
             factory: offline
         cash_on_delivery:
             factory: offline
+        paypal_express_checkout:
+            factory: paypal_express_checkout
+            payum.http_client: @sylius.payum.http_client
+            username: EDITME
+            password: EDITME
+            signature: EDITME
+            sandbox: true
+        credit_card:
+            factory: stripe_checkout
+            secret_key: EDITME
+            publishable_key: EDITME
+            payum.template.layout: SyliusShopBundle::Checkout/layout.html.twig
+            payum.template.obtain_token: SyliusPayumBundle::Action/obtainCheckoutToken.html.twig
```

Comment:
Why configuring it in main config? Default config should contain 0 gateways (except offline). These should be configured in test env only and devs should configure these per project for prod/dev.